### PR TITLE
Added skip map mode to print w/o extent selection & choosing template

### PR DESCRIPTION
**Pull request by hwbllmnn**
This can be used for custom print requirements such as printing a 'normal' pdf without a map and a given template. It skips the app selection and disables the extent rectangle if the config option `skipMapMode` is set.

@terrestris/devs please review

### CODE REVIEW

File: src/view/form/Print.js [hunk 1]
```diff
@@ -335,7 +339,10 @@ Ext.define('BasiGX.view.form.Print', {
         Ext.each(fieldsets, function(fs) {
             var name = fs.name;
             // TODO double check when rotated
-            var featureBbox = fs.extentFeature.getGeometry().getExtent();
+            var featureBbox;
+            if (fs.extentFeature) {
+                fs.extentFeature.getGeometry().getExtent();
```
Comment (weskamm) [inline src/view/form/Print.js]: is this missing a variable assignment?

**Comment by weskamm** [comment]
One comment added. Maybe you can add a short explanation in the code what `skipMapMode` does. Else good to go

File: src/view/form/Print.js (hunk 1 above)
Comment (hwbllmnn) [reply src/view/form/Print.js]: Nice catch, fixed!

**Comment by hwbllmnn** [comment]
Done, description added.

**Pull request merged**